Author: Michael R. Crusoe <crusoe@debian.org>
Description: cloudflare_zlib is not supported by Debian's flate2 package
Forwarded: not-needed
--- bgzip.orig/Cargo.toml
+++ bgzip/Cargo.toml
@@ -77,10 +77,6 @@
 version = "3.3"
 
 [features]
-cloudflare_zlib = [
-    "flate2/cloudflare_zlib",
-    "flate2",
-]
 default = [
     "rust_backend",
     "log",
--- bgzip.orig/Cargo.toml.orig
+++ bgzip/Cargo.toml.orig
@@ -19,7 +19,6 @@
 zlib = ["flate2/zlib", "flate2"]
 zlib-ng-compat = ["flate2/zlib-ng-compat", "flate2"]
 zlib-ng = ["flate2/zlib-ng", "flate2"]
-cloudflare_zlib = ["flate2/cloudflare_zlib", "flate2"]
 libdeflater = ["dep:libdeflater"]
 rayon = ["dep:rayon"]
 log  = ["dep:log"]
